docs/index.md

```
@@ -4,6 +4,20 @@ This is the Python-based OpenML REST API server.
 It's a rewrite of our [old backend](http://github.com/openml/openml) built with a
 modern Python-based stack.
 
+### What's changed?
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)